.projector/workItemTemplates/engineering-fundamentals.yml

```diff
@@ -496,8 +496,8 @@ items:
         acceptanceCriteria: []
         children:
           - name:
-              As a dev crew, we have provided reproducible, actionable, specific and
-              goal-oriented CSE feedback on any blockers or potential
+              As an ISE team, we have provided reproducible, actionable, specific and
+              goal-oriented ISE feedback on any blockers or potential
```

Comment:
this part still says ISE feedback, should be CSE feedback or removed

Reply:
Reset to CSE feedback